### PR TITLE
A lot of changes done: History, Program, Student Seminar, Outreach

### PR DESCRIPTION
**Pull request by krishnapandeykp**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h2 class="d-none d-lg-block align-self-center col-6 text-body-emphasis" style="
       </ng-container>
 
         <li class="nav-item">
-          <a class="nav-link" onclick="window.open('https://cefirse.ird.fr')" href="">IndoFrench Cell</a>
+          <a class="nav-link" onclick="window.open('http://cefirse.ird.fr')" href="">IndoFrench Cell</a>
```
Comment (ICWaR) [inline src/app/app.component.html]: Explain this

File: src/app/app.component.html [hunk 2]
```diff
@@ -4,15 +4,15 @@
     <button class="bg-primary-subtle navbar-toggler btn float-right" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
       <span class="nav-menu-button "></span>
     </button>
-    <a class="d-lg-none d-sm-grid" href="#"><img width="150px" src="assets/images/logo-icwar.png" class="shadow border border-2 border-light " style="border-radius: 5px;"></a>
+    <a class="d-lg-none d-sm-grid" href="#"><img width="150px" src="assets/images/logo-icwar.png" alt="" class="shadow border border-2 border-light " style="border-radius: 5px;"></a>
```
Comment (ICWaR) [inline src/app/app.component.html]: Explain these 3

File: src/app/app.module.ts [hunk 3]
```diff
@@ -51,7 +54,10 @@ import { TimelineComponent } from './views/timeline/timeline.component';
     ActivitiesAndEventsComponent,
     ProgramsComponent,
     StudentsCornerComponent,
-    TimelineComponent
+    TimelineComponent,
+    TestimonialsComponent,
+    OutreachComponent,
+    StudentsSeminarComponent,
```
Comment (ICWaR) [inline src/app/app.module.ts]: Remove comma from last line

File: src/app/app.component.html (hunk 1 above)
Comment (krishnapandeykp) [reply src/app/app.component.html]: Link was not functional, so changed it.

File: src/app/app.component.html (hunk 2 above)
Comment (krishnapandeykp) [reply src/app/app.component.html]: As I was inside the same file to change the URL, saw those errors so tried fixing it, Although, it serves no purpose.

Comment (ICWaR) [reply src/app/app.component.html]: Please revert these changes in that case

File: src/app/app.component.html (hunk 1 above)
Comment (ICWaR) [reply src/app/app.component.html]: http and https will not cause any issue in functionality. I have cross-verified the link is functional.
Using http is a security issue. Please revert this change

File: src/app/views/outreach/outreach.component.html [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+    <iframe class="youtube-player" width="100%" height="350vh" src="https://www.youtube.com/embed/yTR90oWjb5M" title="Floods in a Changing Climate | Dr P P Mujumdar | October 26, 2020" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
   </div>
-  <div class="col-md-6">
+  <div class="col-md-6 col-sm-12 ">
     <h2>Vulnerability of Water Resources to Climate Change</h2><br>
-    <iframe width="600" height="350" src="https://www.youtube.com/embed/2-tiGpm4Pao" title="iSEE Congress 2015 — Session 1: &quot;Vulnerability of Water Resources to Climate Change&quot;" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+    <iframe class="youtube-player" width="100%" height="350vh" src="https://www.youtube.com/embed/2-tiGpm4Pao" title="iSEE Congress 2015 — Session 1: &quot;Vulnerability of Water Resources to Climate Change&quot;" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (ICWaR) [inline src/app/views/outreach/outreach.component.html]: Height: 350vh,
Please go through the meaning of vh usage once and try to understand the meaning generated here

**Review by ICWaR — APPROVED** [review]

**Pull request merged**